sysbuild/CMakeLists.txt

```diff
@@ -557,15 +561,6 @@ function(${SYSBUILD_CURRENT_MODULE_NAME}_pre_cmake)
     endif()
   endif()
 
-  if(SB_CONFIG_SUIT_BUILD_RECOVERY)
```

Comment:
After deleting this part appropriate `select` statements should be added in the `Kconfig.suit` file

Reply:
I think changes in `sysbuild/suit_provisioning/Kconfig.nrf54h20` should be sufficient.. aren't they?